frontend/packages/dev-console/src/components/gitops/GitOpsListItem.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Grid, GridItem, Card, CardBody } from '@patternfly/react-core';
+import { ResourceIcon, history } from '@console/internal/components/utils';
+import { GitOpsAppGroupData } from './gitops-types';
+import './GitOpsList.scss';
+
+interface GitOpsListItemProps {
+  appGroup: GitOpsAppGroupData;
+}
+
+const GitOpsListItem: React.FC<GitOpsListItemProps> = ({ appGroup }) => {
+  const handleCardClick = () => {
+    history.push(`/gitops/application/${appGroup?.name}?url=${appGroup?.repo_url}`);
```

Comment:
`appGroup` will always be defined as per the prop type.

```suggestion
    history.push(`/gitops/application/${appGroup?.name}?url=${appGroup.repo_url}`);
```